data/common.yaml

```diff
@@ -9,6 +9,7 @@ elasticsearch::api_port: 9200
 elasticsearch::api_protocol: http
 elasticsearch::api_timeout: 10
 elasticsearch::autoupgrade: false
+elasticsearch::component_templates: {}
```

Comment:
ah another tiny thing. could you please not add new datat to the common.yaml but directly to the init.pp? That enables puppet-string to pickup the default value. That's our new best practice at vox pupuli, we just had no time yet to migrate all common.yaml files to init.pp

Reply:
Done, I thought PDK complains about this pattern, but for me is ok ;)